ui/pages/create-account/connect-hardware/index.js

```diff
@@ -72,44 +67,16 @@ class ConnectHardwareForm extends Component {
     device: null,
   };
 
-  UNSAFE_componentWillReceiveProps(nextProps) {
-    const { accounts } = nextProps;
-    const newAccounts = this.state.accounts.map((a) => {
-      const normalizedAddress = a.address.toLowerCase();
-      const balanceValue = accounts[normalizedAddress]?.balance || null;
-      a.balance = balanceValue ? formatBalance(balanceValue, 6) : '...';
-      return a;
-    });
-    this.setState({ accounts: newAccounts });
-  }
-
-  componentDidMount() {
-    this.checkIfUnlocked();
-  }
-
-  async checkIfUnlocked() {
-    for (const device of [
-      HardwareDeviceNames.trezor,
-      HardwareDeviceNames.ledger,
-      HardwareDeviceNames.lattice,
-    ]) {
```

Comment:
Removed as we don't have a scenario where we need to check that all devices are unlocked and then subsequently fetch the first page of their accounts.